examples/manifestwork/README.md

```
@@ -15,100 +17,224 @@ workClient, err := grpcsource.NewMaestroGRPCSourceWorkClient(
-		Limit: 1000,
-    Continue: workList.ListMeta.Continue,
+	Limit: 1000,
+	Continue: workList.ListMeta.Continue,
 })
 if err != nil {
   log.Fatal(err)
 }
 
-// Third list: we also list works with last returned `ListMeta.Continue`, this will return
-// all remaining works (500) and the `workList.ListMeta.Continue` will be empty
+// Third page: returns remaining 500 works, Continue will be empty
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)